ct/zabbix.sh

```diff
@@ -57,11 +57,31 @@ header_info
 check_container_storage
 check_container_resources
 if [[ ! -f /etc/zabbix/zabbix_server.conf ]]; then msg_error "No ${APP} Installation Found!"; exit; fi
+msg_info "Stopping ${APP} Services"
+systemctl stop zabbix-server zabbix-agent2
+msg_ok "Stopped ${APP} Services"
+
 msg_info "Updating $APP LXC"
+mkdir -p /opt/zabbix-backup/
+cp /etc/zabbix/zabbix_server.conf /opt/zabbix-backup/
+cp /etc/apache2/conf-enabled/zabbix.conf /opt/zabbix-backup/
+cp -R /usr/share/zabbix/ /opt/zabbix-backup/
+cp -R /usr/share/zabbix-* /opt/zabbix-backup/
+cd /tmp
+wget -q https://repo.zabbix.com/zabbix/7.0/debian/pool/main/z/zabbix-release/zabbix-release_latest+debian12_all.deb
```

Comment:
https://www.zabbix.com/documentation/current/en/manual/installation/upgrade/packages/debian_ubuntu#update-repository-configuration-package

`rm -Rf /etc/apt/sources.list.d/zabbix.list` is this step not needed?

Reply:
I have included it as an option, it probably shouldn't matter. At least it worked without and with the same. Presumably this is there for major updates, e.g. from 7.0 to 8.0 or something like that.